app/assets/javascripts/angular/controller/forms/listForms.js.erb

```diff
@@ -42,20 +26,23 @@ app.controller('ListFormsCtrl', ['$scope', '$http','$auth','$location','$rootSco
     }
   };
 
-  //Modal functionalities
-  $scope.show = function() {
-    ModalService.showModal({
-        templateUrl: "<%= asset_path('forms/modalCreateForm.html') %>",
-        controller: "ListFormsCtrl"
-    }).then(function(modal) {
-        modal.element.modal();
-        modal.close.then(function(result) {x  
-        });
+  //Modal 
+  $scope.animationsEnabled = true;
+
+  $scope.open = function() {
+
+    var modalInstance = $modal.open({
+      animation: $scope.animationsEnabled,
+      templateUrl: "<%= asset_path('forms/modalCreateForms.html') %>",
+      controller: 'ModalInstanceCtrl'
     });
+
   };
 
-  $scope.close = function() {
-    close(500); 
+  $scope.toggleAnimation = function () {
```

Comment:
Do you understand what this function does? Why do you want to allow the user to choose whether or not there will be animations for modals?